test/message/test_runner_output.out

```diff
@@ -197,13 +219,19 @@ not ok 21 - subtest sync throw fail
   error: '1 subtest failed'
   code: 'ERR_TEST_FAILURE'
   ...
+# Subtest: sync throw non-error fail
 not ok 22 - sync throw non-error fail
   ---
   duration_ms: *
   failureType: 'testCodeFailure'
   error: 'Symbol(thrown symbol from sync throw non-error fail)'
   code: 'ERR_TEST_FAILURE'
   ...
+# Subtest: level 0a
+    # Subtest: level 1a
+    # Subtest: level 1b
+    # Subtest: level 1c
+    # Subtest: level 1d
```

Comment:
This doesn't seem correct to me. Should all of these subtest headers be displayed without the actual test output?

Reply:
Thanks. fixed that